src/components/card/card.stories.tsx

```diff
@@ -0,0 +1,339 @@
+import React from 'react'
```

Comment:
nitpicky, but can we keep file name casing consistent? so `Card.stories.tsx`

Reply:
@eamahanna can you take a look at this again? To rename files with git you may have to use `git mv` to get the renaming tracked